bundles/org.openhab.binding.groupepsa/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,551 @@
+		<label>Various</label>
+		<description>Miscellaneous information</description>
+		<channels>
+			<channel id="lastUpdated" typeId="lastUpdated"/>
+			<channel id="privacy" typeId="privacy"/>
+			<channel id="belt" typeId="belt"/>
+			<channel id="emergency" typeId="emergency"/>
+			<channel id="service" typeId="service"/>
+			<channel id="preconditioning" typeId="preconditioning"/>
+			<channel id="preconditioningFailure" typeId="preconditioningFailure"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="current">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>Current</label>
+		<description>Electrical current</description>
+		<category>Energy</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="voltage">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Voltage</label>
+		<description>Voltage</description>
+		<category>Energy</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Temperature</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="daytime">
+		<item-type>Contact</item-type>
+		<label>Daytime</label>
+		<description>Enabled if it is daytime</description>
+		<category>Sun</category>
+	</channel-type>
+
+	<channel-type id="doorLock">
+		<item-type>String</item-type>
+		<label>Door Lock</label>
+		<description>Door lock state</description>
+		<category>Door</category>
+		<state readOnly="true">
+			<options>
+				<option value="Unlocked">Unlocked</option>
+				<option value="Locked">Locked</option>
+				<option value="SuperLocked">Super Locked</option>
+				<option value="DriverDoorUnlocked">Driver Door Unlocked</option>
+				<option value="CabinDoorsUnlocked">Cabin Doors Unlocked</option>
+				<option value="CargoDoorsLocked">Cargo Doors Locked</option>
+				<option value="CargoDoorsUnlocked">Cargo Doors Unlocked</option>
+				<option value="RearDoorsLocked">Rear Doors Locked</option>
+				<option value="RearDoorsUnlocked">Rear Doors Unlocked</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="doorOpen">
+		<item-type>Contact</item-type>
+		<label>Door Open</label>
+		<description>Door is open</description>
+		<category>Door</category>
+	</channel-type>
+
+	<channel-type id="ignition">
+		<item-type>String</item-type>
+		<label>Ignition</label>
+		<description>Ignition state</description>
+		<category>Motion</category>
+		<state readOnly="true">
+			<options>
+				<option value="Stop">Stop</option>
+				<option value="StartUp">Start up</option>
+				<option value="Start">Start</option>
+				<option value="Free">Free</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="moving">
+		<item-type>Contact</item-type>
+		<label>Moving</label>
+		<description>Vehicle is moving</description>
+		<category>Motion</category>
```

Comment:
You don't need to put a category forcibly to any Channel. If there's no suitable category, better leave it away.

Reply:
Ok, will only leave clearly relevant ones in.